core/signal.rbs

```diff
@@ -89,6 +89,12 @@ module Signal
   #     Child died
   #     Terminating: 27460
   #
-  def self.trap: (Integer | String | Symbol signal, ?untyped command) -> (String | Proc)
-               | (Integer | String | Symbol signal) { (Integer arg0) -> untyped } -> (String | Proc)
+  def self?.trap: (Integer | interned signal) { (Integer) -> void } -> trap_command
```

Comment:
Does it always return `Proc`?

Reply:
It returns whatever the previously registered `trap` command was.